pkg/crc/oc/oc.go

```diff
@@ -39,22 +41,27 @@ func (oc OcLocalRunner) GetKubeconfigPath() string {
 	return oc.KubeconfigPath
 }
 
+func (oc OcLocalRunner) GetContext() string {
+	return oc.Context
+}
+
 // UseOcWithConfig return the oc binary along with valid kubeconfig
 func UseOCWithConfig(machineName string) OcConfig {
 	localRunner := OcLocalRunner{
 		OcBinaryPath:   filepath.Join(constants.CrcBinDir, constants.OcBinaryName),
 		KubeconfigPath: filepath.Join(constants.MachineInstanceDir, machineName, "kubeconfig"),
+		Context:        constants.DefaultContext,
 	}
 	return NewOcConfig(localRunner)
 }
 
 func (oc OcConfig) RunOcCommand(args ...string) (string, string, error) {
-	args = append(args, "--kubeconfig", oc.runner.GetKubeconfigPath())
+	args = append(args, "--kubeconfig", oc.runner.GetKubeconfigPath(), "--context", oc.runner.GetContext())
```

Comment:
I believe we also want to add `--cluster crc` as this:
```
$ oc config get-contexts  --kubeconfig ~/.crc/cache/<bundle>/kubeconfig
CURRENT   NAME                                      CLUSTER                AUTHINFO                          NAMESPACE
          admin                                     crc                    admin
          default/api-crc-testing:6443/kube:admin   api-crc-testing:6443   kube:admin/api-crc-testing:6443   default
*         kind-kind                                 kind-kind              kind-kind
```
gives me the feeling that we could have an `admin` context in the `kind-kind` namespace/cluster too?